common/swagger/swagger.yaml

```diff
@@ -64,6 +64,25 @@ paths:
           description: Successful operation
           schema:
             $ref: '#/definitions/ScoreRequest'
+  '/model/features':
```

Comment:
@Rajimut @orendain nitpick - name "features" is often used for referring the data attributes. 
I would better to use "capabilities".

Reply:
Great catch, thanks!  Fixed.